src/ui/Facet/FacetValue.ts

```diff
@@ -0,0 +1,123 @@
+import { Assert } from '../../misc/Assert';
```

Comment:
The `FacetValue` class used to be in the `FacetValues.ts`. I extracted it into it's own file to avoid introducing a circular dependency, but otherwise it is unchanged.